index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio Website</title>
+</head>
+<body>
+    <header>
+        <div class="logo">
+          <a>My Logo</a>
+        </div>
+        <div class="nav-bar">
+            <img src="image/Vector.png" alt="Nav Bar" class="navImg1">
+            <img src="image/Vector.png" alt="Nav Bar" class="navImg2">
+            <img src="image/Vector.png" alt="Nav Bar" class="navImg3">
+
+        </div>
+    </header>
+    <main>
+       <section>
+           <div class="Backgroud">
+  
+    
+           </div>
+        <div class="title-name">
+            <h2>I’m Amol
+                Glad to see you!
+            </h2>
```

Comment:
- Please make this an `<h1>` to make your HTML semantically correct.
- Please set the font size to `40px` as in the Figma design, and also use a line break to break words accordingly.
##### Figma design
![Screenshot from 2022-05-28 08-08-05](https://user-images.githubusercontent.com/58771507/170814721-5ece6a2b-994d-4491-bcf6-163d290265f2.png)

##### Your design
![Screenshot from 2022-05-28 08-08-56](https://user-images.githubusercontent.com/58771507/170814750-13eba8d8-8a89-44a0-b38e-fa64cc253d5e.png)